src/app/electra/reducer.ts

```diff
@@ -27,6 +27,23 @@ export default function electraReducer(state = initialState, action: any) {
         ...state
       }
     }
+    case ActionNames.GET_GLOBAL_BALANCE_SUCCESS: {
+      return {
+        ...state,
+        electraJs: {
+          ...state.electraJs,
+          wallet: {
+            ...state.electraJs.wallet,
+            GLOBAL_BALANCE: action.globalBalance
```

Comment:
I think you only need to set the state of GLOBAL_BALANCE.

So instead of:
```
return {
        ...state,
        electraJs: {
          ...state.electraJs,
          wallet: {
            ...state.electraJs.wallet,
            GLOBAL_BALANCE: action.globalBalance
          }
        }
      }
    }
```
You can have
```
return {
        ...state,
          wallet: {
            GLOBAL_BALANCE: action.globalBalance
          }
        }
      }
    }
```